MVCForum/MVCForum.Website/Installer/en-GB.csv

```diff
@@ -160,7 +160,7 @@ LogOn.LogOff,Log Off
 LogOn.LogOn,Log In
 LogOn.Button.Text,Log In
 LogOn.Error.UnableToLogIn.Text, Unable to log in. Please check your details.
-LogOn.Introduction.Text, Log in text TBC
+LogOn.Introduction.Text, Collaborate, connect, share and learn
```

Comment:
Lets hard code these, language strings are really going to work going forward. make this change in logon.cshtml and replace  @Html.LanguageString("LogOn.Introduction.Text") with Collaborate, connect, share and learn. You'll need to do the same for Logon.Button.Text etc etc